osm2gtfs/core/cache.py

```diff
@@ -37,3 +37,30 @@ def read_data(name):
             return content
         else:
             return {}
+
+    @staticmethod
+    def write_file(name, content):
+        """Function to write cache
+
+        Writes an object (content) with an indicated name to a file on the
+        hard drive.
+
+        """
+        if not os.path.isdir('data'):  # checking is the dir exist
```

Comment:
there's a typo in the comment, but the method name would be clear enough for my taste.